src/main/java/tc/oc/pgm/PGMImpl.java

```diff
@@ -249,6 +250,8 @@ public void onEnable() {
       return;
     }
 
+    FlairUtils.setMatchManager(matchManager);
```

Comment:
This isn't going to work, also for name renderer.

Instead, there should be a `PGM.getFlairRegistry()`